docs/source/chainermn/model_parallel/example4_ensemble.rst

```diff
@@ -0,0 +1,43 @@
+Example 4: Ensemble
+===================
+
+Averaging ensemble is one application which collective communications can effectively be applied to.
```

Comment:
I think an additional sentence that describes what and why ensemble is good would make this use case more intuitive.